public/api/conf/1.0/application.yaml

```diff
@@ -821,14 +846,15 @@ components:
         startDate:
           pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}$
           type: string
-          description: Date in the format YYYY-MM-DD in the range 1900-01-01 to 2099-12-31 accommodating leap years
+          description: Start date of the group's period of account, which cannot be before 2016-10-01
         endDate:
           pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}$
           type: string
-          description: Date in the format YYYY-MM-DD in the range 1900-01-01 to 2099-12-31 accommodating leap years
-      description: Period of account
+          description: End date of the group's period of account, which must be within 18 months of the start date
+      description: The worldwide group's period of account
```

Comment:
I intentionally added this for as the only difference was "The" so instead of having two descriptions, I used this one to avoid duplication. This was also intentional